src/Workspaces/Remote/ServiceHub/Services/ProcessTelemetry/RemoteProcessTelemetryService.PerformanceReporter.cs

```diff
@@ -63,39 +58,26 @@ protected override void OnPaused()
 
             protected override Task ExecuteAsync()
             {
-                using (var pooledObject = SharedPools.Default<List<ExpensiveAnalyzerInfo>>().GetPooledObject())
                 using (RoslynLogger.LogBlock(FunctionId.Diagnostics_GeneratePerformaceReport, CancellationToken))
```

Comment:
📝 This should also return early if telemetry is disabled

Reply:
Done